lib/http-context.js

```diff
@@ -456,6 +465,11 @@ function toXML(input) {
       }
     });
   }
+  if (returns.xml && returns.xml.wrapperElement === false) {
+    //allowing XML output to have no root element
+    var match = xml.match(/<response>([\s\S]*?)<\/response>$/);
+    xml = match[1].replace(/^[\s]{0,2}/gim, '').trim();
+  }
```

Comment:
@bajtos the current plugin we are using to parse javascript to XML does not allow JSON objects with no [`root element`](https://github.com/michaelkourlas/node-js2xmlparser#usage), do you have any suggestion on how to approach this? Using a different plugin will cause the way of attributes handling to be different.